### DISCUSSION TITLE
make a discord server for scratchconnect

### DISCUSSION THREAD

**Discussion by Felixoofed** [Sid72020123/scratchconnect#28] [Ideas]
please

**Comment by aidaner** [comment]
why?

**Reply by Felixoofed** [reply]
i dont know it seems useful to get help that way
and/or to just talk about scratchconnect things

**Reply by Quantum-Codes** [reply]
Ask here. Making a discord server for maybe 10 people who actually use Scratch and aren't underage (scratch's audience is 7+ ig. Not 13+)

**Reply by Sid72020123** [reply]
I don't have any plans to make a discord server yet...
I think I will make it when the library has more new features and a good documentation.
Sorry for replying late. I am busy because of school.

**Reply by noverd** [reply]
What is Discord?

**Reply by Quantum-Codes** [reply]
Unmoderated chatting app

**Reply by noverd** [reply]
Ok!

**Comment by UniqueName12345** [comment]
Wouldn't this make scratchconnect unlinkable on scratch?

**Reply by Sid72020123** [reply]
Yes. I won't share the invite link in the docs.
I would share it with people on that platform...

**Reply by UniqueName12345** [reply]
Ah! Well, in that case...
